src/_versions.ts

```diff
@@ -11,8 +11,8 @@ export interface TsAppVersion {
 export const versions: TsAppVersion = {
     version: '0.1.1',
     name: 'finesse-frontend',
-    versionDate: '2023-11-06T17:26:41.959Z',
-    gitCommitHash: '499e4ad',
-    versionLong: '0.1.0-499e4ad',
+    versionDate: '2023-11-24T21:13:58.324Z',
+    gitCommitHash: '6924943e',
+    versionLong: '0.1.0-6924943e',
```

Comment:
versionLong is 0.1.0 but version is 0.1.1?

Reply:
this is supposed to be generated, we can't have the version going back.

Reply:
I changed the version to 0.2.0 this is what it is on main